LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2017 Boston University, the Lighting Enabled Systems and 
```

Comment:
Replace with:

+Copyright (c) 2017 Boston University, the Lighting Enabled Systems and 
 +Applications (LESA) Engineering Research Center (ERC), and other SinglePixel contributors. 
 +All rights reserved.